news/streaming

```diff
@@ -0,0 +1,24 @@
+### Enhancements
+
+* Based on conda-package-streaming >= 0.5.0 instead of libarchive.
+* Requires the `python-zstandard` (`zstandard`) library.
+* Threadsafe `extract()` function.
+* More efficient `.conda` handling.
+
+### Bug fixes
+
+* <news item>
+
+### Deprecations
+
+* Remove broken `verify` subcommand.
+* Remove support for `binsort` (was supposed to help with `tar.bz2`
+  compression). (Use `.conda` instead.)
+
+### Docs
```

Comment:
If you added a Sphinx site, wouldn't that warrant at least a mention in the release documentation? 😅

Reply:
It is the documentation.